sigstore-public-deployment-spec.md

```diff
@@ -133,28 +133,28 @@ Sigstore does not operate a timestamp authority at this time. We do include trus
+The root-of-trust data can change over time due to scheduled key rotations, log sharding, or compromise. Rather than tying a specific release of a client to a specific set of keys (which would require an upgrade of the client in order to trust new signatures), Sigstore distributes it using [The Update Framework](https://theupdateframework.io/) (TUF).
 
 The resources for the Sigstore Roots-of-Trust are managed in the [sigstore/root-signing repository](https://github.com/sigstore/root-signing). To increase trust, the TUF repository is managed in a transparent and open way, allowing community members to inspect and verify all steps and updates This helps to detect any malicious activity, and ensures that users can trust the repository. When updates to the TUF repository are made, Sigstore community members are encouraged to verify all the steps before publishing a new version.
 
 ### 4.1. The Update Framework
 
 [The Update Framework](https://theupdateframework.io/) (TUF) is a framework designed for software update systems, protecting against a number of subtle attacks (such as rollback attacks, in which an attacker serves users a formerly-valid artifact). Its [specification](https://theupdateframework.github.io/specification/latest/) is general enough for arbitrary digital artifacts. Sigstore uses TUF to distribute the public keys and certificate chains for its services, rather than a web PKI-inspired solution, because TUF offers better security against specific attacks on the distribution of key material,vastly simplifies revocations, and is feasible at the scale of a repository of key materials.
 
-In Sigstore’s use of TUF, clients ship with a TUF trust root (which may be rotated over time). From the TUF trust root, clients can derive trust in the public keys of the Sigstore instance (the full details follow the TUF [specification](https://theupdateframework.github.io/specification/latest/) and are out-of-scope for this document).
+In Sigstore’s use of TUF, clients ship with an initial TUF root (which may be rotated over time). This allows the client to securely download current
+root-of-trust files from the Sigstore instances TUF repository (the full details follow the TUF [specification](https://theupdateframework.github.io/specification/latest/) and are out-of-scope for this document).
 
-#### 4.1.1 Artifacts
+#### 4.1.1 Artifacts in the TUF repository
 
-* Public keys for the certificate transparency logs, e.g. Ctfe.pub
-* Certificate chains for the CA (Fulcio), e.g. Fulcio\_v1.crt.pem
-* Public keys for the signature transparency log, e.g. rekor.pub
-* Public key used to sign the release artifacts: artifact.pub
+* `trusted_root.json` (in future `trusted_root.v<MAJOR>.<MINOR>.json`)
+* `signing_config.json` (in future `signing_config.v<MAJOR>.<MINOR>.json`)
```

Comment:
@loosebazooka to follow up on target discovery

Reply:
If it wasn't clear: From my perspective the targetpaths are magic strings. @kommendorkapten wanted to the version numbering scheme documented here and that's fine by me, but each targetpath is still a magic string that client must know and should not try to "discover".

Reply:
Thanks for clarifying!
Sorry for the comment without context. We were discussing whether we needed target discovery, this seems much simpler.